cypress/integration/pages/liveRadio/testsForCanonicalOnly.js

```diff
@@ -44,6 +50,49 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
         });
       }
     });
+
+    describe('Radio Schedule', () => {
+      it('should be displayed if there is enough schedule data', () => {
+        const isRadioScheduleOnPage = serviceHasRadioSchedule({
+          service,
+          variant,
+        });
+        cy.log(
+          `Live Radio Page configured for Radio Schedule? ${isRadioScheduleOnPage}`,
+        );
+
+        if (isRadioScheduleOnPage) {
+          // Override is being added twice causing a false negative on the call to the schedule! I think other tests need the override though. Will look at this again
+          const liveRadioPath = Cypress.env('currentPath').replace(
+            '?renderer_env=live',
+            '',
+          );
+          cy.request(getRadioScheduleEndpoint(liveRadioPath)).then(
+            ({ body: scheduleJson }) => {
+              const { schedules } = scheduleJson;
+
+              const isRadioScheduleDataComplete = isRadioScheduleComplete(
+                schedules,
+              );
+              cy.log(
+                `Radio Schedule is displayed? ${isRadioScheduleDataComplete}`,
+              );
+              if (isRadioScheduleOnPage && isRadioScheduleDataComplete) {
+                cy.log('Schedule has enough data');
+                cy.get('div[class*="RadioScheduleSection"]').should('exist');
```

Comment:
The integration tests added an attribute to the RadioScheduleWrapper: `data-e2e="radio-schedule"` - this could be a better way of selecting the element than the class name selector as it's less likely to ever be changed. 

```suggestion
                cy.get('div[data-e2e="radio-schedule"]').should('exist');
```

Reply:
These nice identifiers were not present when I was writing the test, but this will be a good improvement, which I will also be using to identify other parts of the schedule in other tests I add.